iam/api-client/src/test/java/iam/snippets/ServiceAccountTests.java

```diff
@@ -90,6 +91,9 @@ public void stage2_testServiceAccountKeyCreate() {
     CreateServiceAccountKey.createKey(PROJECT_ID, SERVICE_ACCOUNT);
     String got = bout.toString();
     assertThat(got, containsString("Created key:"));
+    String serviceAccountKeyPath = got.substring(got.lastIndexOf(":") + 1);
+    SERVICE_ACCOUNT_KEY = serviceAccountKeyPath
```

Comment:
are these tests guaranteed to run in order? I think it might be better to do this in the other test, or combine them.

Reply:
These tests will run in the lexicographic order as specified in line number 35. The tests are ordered around that principle.
Hence I am assuming that the order should work!